source/agora/node/FullNode.d

```diff
@@ -533,37 +450,15 @@ public class Node : API
         this.enroll_man.getValidatorPreimage(enroll_key, preimage);
         return preimage;
     }
-}
-
-
-/*******************************************************************************
-
-    Boots up a node that listen for network requests and blockchain data
-
-    This is called from the main or CLI.
-    The initialization process of the node is then completed.
-
-    Params:
-      config = A parsed and validated config file
 
-*******************************************************************************/
-
-public Node runNode (Config config)
-{
-    Log.root.level(config.logging.log_level, true);
-    log.trace("Config is: {}", config);
-
-    auto settings = new HTTPServerSettings(config.node.address);
-    settings.port = config.node.port;
-    auto router = new URLRouter();
+    /***************************************************************************
 
-    mkdirRecurse(config.node.data_dir);
+        Called when a transaction was accepted into the transaction pool.
+        This is a no-op for FullNode. A Validator node overrides this behavior.
 
-    auto node = new Node(config);
-    router.registerRestInterface(node);
-    node.start();  // asynchronous
+    ***************************************************************************/
 
-    log.info("About to listen to HTTP: {}", settings.port);
-    listenHTTP(settings, router);
-    return node;
+    protected void onAcceptedTransaction () @safe
+    {
+    }
 }
```

Comment:
Moving this could be its own commit

Reply:
The existing code is `this.nominator.tryNominate()` in `FullNode.putTransaction`
Because the Nominator has been moved from the corresponding refectoring, it must be handled by the same commit.

Reply:
Oh, I understood the point.
I separated it with a separate commit.